module/solutions/helper.js

```diff
@@ -344,177 +582,256 @@ module.exports = class SolutionsHelper {
    * @returns {JSON} - scope in solution.
    */
 
-  static setScope(solutionId, scopeData) {
+  static setScope(programId, solutionId, scopeData) {
```

Comment:
@praveenKDass if new param is introduced at first. will it affect other calls of same function? please check , where ever this function is called param should be passed accordingly.

Reply:
Yes vishnu handled that @VISHNUDAS-tunerlabs